Tools/CISupport/build-webkit-org/config.json

```diff
@@ -441,6 +441,7 @@
                   {
                     "name": "GTK-Linux-64-bit-Debug-Tests", "factory": "TestAllButJSCFactory",
                     "platform": "gtk", "configuration": "debug", "architectures": ["x86_64"],
+                    "additionalArguments": ["--exit-after-n-crashes-or-timeouts=100"],
```

Comment:
Maybe 200 or 300 is even better

Reply:
Currently some builds finish correctly even with the default 50 limit, so I think 100 should be enough.